libs/sdk-ui-charts/src/highcharts/chartTypes/_util/common.ts

```diff
@@ -166,8 +166,26 @@ export const unwrap = (wrappedObject: unknown): any => {
     return wrappedObject[Object.keys(wrappedObject)[0]];
 };
 
-export function percentFormatter(value: number): string {
-    return isNil(value) ? "" : `${parseFloat(value.toFixed(2))}%`;
+const getNumberWithGivenDecimals = (value: number, decimalNumbers: number) =>
+    decimalNumbers === 0 ? `${Math.round(value)}%` : `${parseFloat(value.toFixed(decimalNumbers))}%`;
+
+export function percentFormatter(value: number, format?: string): string {
+    let numberOfDecimals = 2;
```

Comment:
This function do two things. Deciding on number of decimals and then it does formatting. IMHO it would be better to have function that will determine number of decimals from format string and then this function that will use it. It will be more clear. 

Also, this way you don't need to use `let` that is more hard to read as you need to track where you change the value. However, this is really opinionated and there is nothing wrong with `let` itself.